rust_snuba/src/types.rs

```diff
@@ -168,7 +169,7 @@ impl BytesInsertBatch {
     }
 
     pub fn len(&self) -> usize {
```

Comment:
I don't know if this suggestion makes sense, but does the naming of this method potentially create the expectation that you're getting the length of the bytes array? e.g. can we prevent this from happening again by renaming this method itself to `pub fn row_count(&self) -> usize`?

Reply:
@onewland potentially? I think in the sequence of changes there wasn't any confusion about what this method is supposed to do. rather, we made a change to how the batch is internally represented, then decided against it, and accidentally left a few changes from the reverted change in. and i didn't review the final diff properly